sdk/servicebus/service-bus/src/core/receiverHelper.ts

```diff
@@ -12,7 +12,7 @@ import * as log from "../log";
  * @ignore
  */
 export class ReceiverHelper {
-  private _isSuspended: boolean = false;
+  private _canReceiveMessages: boolean = false;
```

Comment:
Nit: I recommend keeping the older name `_isSuspended` for this private boolean tracker. The name `canReceiveMessages` for the public getter is good. It will include both the "suspension" status and the validity of the receiver

Reply:
Done. :)